crates/sui-core/src/execution_engine.rs

```diff
@@ -640,6 +640,65 @@ fn transfer_sui<S>(
     Ok(())
 }
 
+pub(crate) fn manual_execute_move_call<
```

Comment:
I kind of expected this function or one like it to be used in both the dry-run path and the normal move-call path to keep the behaviour between those two in sync as much as possible.  I don't think that's what's happening here, but is that possible?

Reply:
This is a helper function for dev_inspect (and  maybe later dry_run) for manually invoking a Move function without having to have a gas object. So the usage will be rather niche (and a tad unsafe)